core/src/components/input/test/label-placement/input.e2e.ts

```diff
@@ -213,4 +210,23 @@ configs({ modes: ['md'], directions: ['ltr'] }).forEach(({ title, screenshot, co
       expect(await input.screenshot()).toMatchSnapshot(screenshot(`input-async-label`));
     });
   });
+  test.describe(title('input: floating/stacked label layering'), () => {
+    test('label should not be covered by text field', async ({ page }) => {
```

Comment:
Let's add a `testInfo.annotations` to track the reason behind this test.

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/27870/commits/9ffc1d933c2d295ed832c23162f0b741679e903e